README.md

```diff
@@ -1,5 +1,8 @@
 # Foundations Course
 
+## Tentative Schedule
```

Comment:
Does your branch have the latest changes from master? This file shouldn't be there.

Reply:
I am sorry, I made a mistake there.

Reply:
Done

Reply:
I think you still haven't merged back from master